src/shared/components/Header/Navigation/component.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Navigation, ToggleMenu, SearchButton } from './styles';
+import { MenuListItem } from '../interfaces';
+import { SearchIcon } from '../../../icons';
+
+type NavigationProps = {
+  menuList: MenuListItem[];
+  onToggle(type: string): void;
+};
+
+const Header: React.FC<NavigationProps> = (props: NavigationProps) => {
+  const onToggleSearchBar = (): void => {
+    props.onToggle('search');
+  };
+
+  return (
+    <Navigation>
+      <ul>
+        <li>
+          <SearchButton onClick={onToggleSearchBar}>
+            <SearchIcon width="12px" height="12px" />
+            Пошук
+          </SearchButton>
+        </li>
+        {props.menuList.map(item => {
+          return (
+            <li key={item.name}>
+              <a href={item.link}>{item.name}</a>
+            </li>
+          );
+        })}
+      </ul>
```

Comment:
Looks like, orange button is missing here 

<img width="708" alt="Screenshot 2020-04-13 at 17 58 39" src="https://user-images.githubusercontent.com/13078761/79131116-bbff2c80-7db0-11ea-8bcc-46704131f4f3.png">

Reply:
as discussed - will be added in terms of separate ticket.